trade-finance-manager-ui/server/controllers/utilisation-reports/find-reports-by-year/index.test.ts

```diff
@@ -262,12 +262,16 @@ describe('controllers/utilisation-reports/find-reports-by-year', () => {
       });
 
       jest.mocked(api.getAllBanks).mockResolvedValue(BANKS);
+      jest.mocked(api.getReportsByBankAndYear).mockResolvedValue({ bankName: BANK_NAME_ONE, year: yearQuery, reports: [] });
 
       // Act
       await getFindReportsByYear(req, res);
 
       // Assert
-      expect(res._getRenderView()).toEqual('utilisation-reports/previous-bank-reports-by-year.njk');
+      expect(res._getRenderView()).toEqual('utilisation-reports/utilisation-reports-by-bank-and-year-results.njk');
+      expect((res._getRenderData() as UtilisationReportsByBankAndYearViewModel)?.bankName).toBe(BANK_NAME_ONE);
+      expect((res._getRenderData() as UtilisationReportsByBankAndYearViewModel)?.year).toBe(yearQuery);
+      expect((res._getRenderData() as UtilisationReportsByBankAndYearViewModel)?.reports).toStrictEqual([]);
```

Comment:
I do think it would be good to check that if reports are returned they get mapped over correctly if it isn't too much of a faff to do so